### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by gestchild**
- don't want to show the iiifPresentationPreview as well as the image preview

### CODE REVIEW

File: common/utils/works.js [hunk 1]
```diff
@@ -40,10 +40,13 @@ export function getProductionDates(work: Work) {
 export function getDownloadOptionsFromManifest(
   iiifManifest: IIIFManifest
 ): IIIFRendering[] {
-  const sequence = iiifManifest.sequences.find(
-    sequence => sequence['@type'] === 'sc:Sequence'
-  );
-  return (sequence && sequence.rendering) || [];
+  const sequence =
+    iiifManifest.sequences &&
```
Comment (jamesgorrie) [inline common/utils/works.js]: Do we not have `sequences` sometimes?

File: common/utils/works.js [hunk 2]
```diff
@@ -40,10 +40,13 @@ export function getProductionDates(work: Work) {
...
+    iiifManifest.sequences.find(
+      sequence => sequence['@type'] === 'sc:Sequence'
+    );
+  // $FlowFixMe
```
Comment (jamesgorrie) [inline common/utils/works.js]: What's the issue here, do we need to update the flowtype?

File: common/utils/works.js [hunk 3]
```diff
@@ -40,10 +40,13 @@ export function getProductionDates(work: Work) {
...
+  return sequence && sequence.rendering ? [sequence.rendering] : [];
```
Comment (davidpmccormick) [inline common/utils/works.js]: I guess this would be the same as `return sequence && [sequence.rendering].filter(Boolean)`?

Comment (jamesgorrie) [reply common/utils/works.js]: Or `return [sequence && sequence.rendering].filter(Boolean);`
💯 🔪 🐱

Comment (jamesgorrie) [reply common/utils/works.js]: saying that, why is this an array?

**Pull request merged**